pages/live/live.js

```diff
@@ -92,6 +93,38 @@ function setup () {
   })
 
   document.querySelector('#transcript-star').addEventListener("click", handleTranscriptStarred)
+
+  // user actions on auto-summary
+  document.querySelector('#auto-summary-copy-button').addEventListener("click", () => {
+    const summary_text = document.querySelector('#auto-summary-container').textContent.trim()
+    if (summary_text.length) {
+      window.navigator.clipboard.writeText(summary_text)
+        .then(() => toast('copied!', 2000, '#3a5', '#fff'))
+        .catch(() => toast('could not copy summary to clipboard', 3000, '#c00', '#fff'))
+    }
+  })
+  document.querySelector('#auto-summary-inject-button').addEventListener("click", () => {
+    const summary_text = document.querySelector('#auto-summary-container').textContent.trim()
+    if (summary_text.length) {
+      chrome.runtime.sendMessage(_extensionId, {
+        'type': MESSAGE_TYPES.EXTERNAL_TO_BACKGROUND_TEXT_SUMMARY,
+        'summary': summary_text,
+      }, (success) => {
+        if (success) {
+          toast('copied!', 2000, '#3a5', '#fff')
```

Comment:
for robustness, you might want to also copy the text to clipboard here (in case we think we can copy it directly but we actualyl cant)

Reply:
I'm not totally sold on the idea of doing this, since users might not be expecting it to clobber their clipboard contents (unless we replace "Copy to dialer notes" with "Copy to dialer notes and clipboard" but then it doesn't fit on one line).

I thiiiink I'd rather try to be more accurate about the return value propagated back from the extension, and come up with a phrasing or UI where the user is ok with doing a fallback option that might be select-ctrlC-ctrlV. This might just mean making the "copy to clipboard" button visible all the time.

But it's in #63 now in case we do want it and I have to hit the "merge" button from my phone in Canada.